lib/fathead/react_native/output.txt

```diff
@@ -0,0 +1,1410 @@
+ActionSheetIOS	A							http://facebook.github.io/react-native/docs/getting-started.html			<section class="prog__container"><p>Display an iOS action sheet. The options object must contain one or more of:</p></section>	https://facebook.github.io/react-native/releases/0.40/docs/actionsheetios.html#
+ActionSheetIOS ActionSheetIOS.showActionSheetWithOptions	R	ActionSheetIOS.ActionSheetIOS.showActionSheetWithOptions										
```

Comment:
@moollaza You're right, ActionSheetIOS should not be duplicated here. I just changed that.

Regarding the last suggested redirect - I would have to implement disambiguations generation, like in the python fathead. Will look into that, if it is easy enough.